src/Symfony/Component/PasswordHasher/Tests/Hasher/NativePasswordHasherTest.php

```diff
@@ -100,6 +100,10 @@ public function testBcryptWithLongPassword()
 
     public function testBcryptWithNulByte()
     {
+        if (\PHP_VERSION_ID >= 80218 && \PHP_VERSION_ID < 80300 || \PHP_VERSION_ID > 80305) {
+            $this->markTestSkipped('password_hash() does not accept plain passwords containing null bytes since PHP 8.2.18 and 8.3.5');
```

Comment:
Shouldn't it be `nul` instead of `null`?

```suggestion
            $this->markTestSkipped('password_hash() does not accept plain passwords containing nul bytes since PHP 8.2.18 and 8.3.5');
```

Reply:
Not sure 🤔 What if we use the wording from the PHP commit instead ("null character")?

Reply:
I might be wrong, but it seems to me that the commit is "wrong". `\0` is the `NUL` car, see https://github.com/php/php-src/blob/72b2524c589bb9e60e1cb58c0c065162811a060a/main/spprintf.c#L100

Reply:
ah makes sense, we even have the changed tests named that way